README.md

```
@@ -1,66 +1,96 @@
+
+### Option 1: Docker (Recommended)
+
+```bash
+git clone https://github.com/zielu92/solobilly.git
+cd solobilly
+cp .env.example .env
+./vendor/bin/sail build --no-cache
+./vendor/bin/sail up
+./vendor/bin/sail artisan migrate --seed
+``` 
+### Option 1: Docker (Recommended)
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)